doc/api/path.md

```diff
@@ -68,7 +68,8 @@ changes:
 * Returns: {string}
 
 The `path.basename()` methods returns the last portion of a `path`, similar to
-the Unix `basename` command.
+the Unix `basename` command. Trailing `/` (and `\\` on Windows) characters are
+ignored.
```

Comment:
How about moving the paragraph

> Note: On Windows, both the forward slash (`/`) and backward slash (`\\`) characters are accepted as path delimiters; however, only the backward slash (`\\`) will be used in return values.

from the very bottom to the section about `path.sep` and referencing that section: "Trailing directory separators are ignored, see [`path.sep`][]"?